docs/usage/help.rst.inc

```diff
@@ -112,11 +112,13 @@ borg help placeholders
 
  {now}
 
-     The current local date and time.
+     The current local date and time, by default in ISO-8601 format.
+     You can also supply your own `format string <https://docs.python.org/3.5/library/datetime.html#strftime-and-strptime-behavior>`_, e.g. {now:%Y-%m-%d_%H:%M:%S}
 
  {utcnow}
 
-     The current UTC date and time.
+     The current UTC date and time, by default in ISO-8601 format.
+     You can also supply your own `format string <https://docs.python.org/3.5/library/datetime.html#strftime-and-strptime-behavior>`_, e.g. {utcnow:%Y-%m-%d_%H:%M:%S}
```

Comment:
btw, it is not strictly necessary to generate the .inc files for a PR - I generate them routinely before doing a release.

esp. if stuff goes into 1.0-maint and gets merged into master, frequent changes in *.inc that sometimes cause merge issues are rather annoying.